modules/heat_conduction/src/actions/ThermalContactAction.C

```diff
@@ -36,8 +48,8 @@ ThermalContactAction::validParams()
       "GapValueAux",
       "A string representing the Moose object that will be used for computing the gap size");
   params.addRequiredParam<NonlinearVariableName>("variable", "The variable for thermal contact");
-  params.addParam<BoundaryName>("primary", "The primary surface");
-  params.addRequiredParam<BoundaryName>("secondary", "The secondary surface");
+  params.addParam<std::vector<BoundaryName>>("primary", "The primary surface");
+  params.addRequiredParam<std::vector<BoundaryName>>("secondary", "The secondary surface");
```

Comment:
change the description to 'The list of boundary IDs referring to primary sidesets'